src/csdap_bulk_download/client.py

```diff
@@ -143,4 +147,4 @@ def login(self, username: str, password: str) -> None:
         response = self._request_auth("/token", method="POST", data={"code": code})
         token = response.json()["access_token"]
 
-        self.session.headers["Authorization"] = f"Bearer: {token}"
```

Comment:
Boy, this was a pain to track down ... **csdap-orders** dealt with the (incorrect) `:`, but **csdap_data_authorization** didn't.